.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        goversion: ["1.18.x", "1.19.x"]
+        goversion: ["1.21.x", "1.22.x"]
```

Comment:
Do we want to bump the go version? (I see we used the `maps.Clone` later down)

Reply:
Yes, we use `maps.Clone` for a permanent fix, so I bumped the versions just to get this rolling quickly